msg/vehicle_command.msg

```diff
@@ -2,63 +2,64 @@
 uint32 VEHICLE_CMD_CUSTOM_0 = 0				# test command
 uint32 VEHICLE_CMD_CUSTOM_1 = 1				# test command
 uint32 VEHICLE_CMD_CUSTOM_2 = 2				# test command
-uint32 VEHICLE_CMD_NAV_WAYPOINT = 16			# Navigate to MISSION. |Hold time in decimal seconds. (ignored by fixed wing, time to stay at MISSION for rotary wing)| Acceptance radius in meters (if the sphere with this radius is hit, the MISSION counts as reached)| 0 to pass through the WP, if > 0 radius in meters to pass by WP. Positive value for clockwise orbit, negative value for counter-clockwise orbit. Allows trajectory control.| Desired yaw angle at MISSION (rotary wing)| Latitude| Longitude| Altitude| 
```

Comment:
No, that's fine. I just needed to understand the background. In general we try to reduce whitespace as much as possible.